astor/code_gen.py

```diff
@@ -50,22 +50,30 @@ def to_source(node, indent_with=' ' * 4, add_line_information=False,
                                 pretty_string)
     generator.visit(node)
     generator.result.append('\n')
-    return pretty_source(str(s) for s in generator.result)
+    if set(generator.result[0]) == '\n':
```

Comment:
I just noticed that this must not be doing anything, because it will never be True.  I removed a conditional in the write function that was seeing if the result list was empty, so thought I had to add this (and did it wrong, obviously), but the output from all the round-trips was fine, so this obviously isn't needed.